### PR TITLE
fix(defaults): remove duplicate default settings and sort them

### PR DESCRIPTION
**Pull request by benox3**

### CODE REVIEW

**Comment by CrossR** [comment]
Thanks for de-duplicating this and fixing it up!

It looks like you are getting a CI failure at the mo since it is failing the parse the config. I think that's due to the final comma in the config causing issues.

File: src/editor/Core/ConfigurationDefaults.re [hunk 1]
```diff
@@ -10,30 +10,26 @@ let getDefaultConfigString = configName =>
...
-  "editor.highlightActiveIndentGuide": true,
   "editor.renderWhitespace": "all",
+  "editor.tabSize": 4,
+  "editor.zenMode.hideTabs": true,
+  "files.exclude": ["_esy", "node_modules"],
   "workbench.activityBar.visible": true,
   "workbench.editor.showTabs": true,
+  "workbench.iconTheme": "vs-seti",
   "workbench.sideBar.visible": true,
   "workbench.statusBar.visible": true,
```
Comment (CrossR) [inline src/editor/Core/ConfigurationDefaults.re]: The parser we are using doesn't like/we've not set it up to like a trailing comma, so I'm assuming this is what is causing the CI failure.

**Comment by benox3** [comment]
> Thanks for de-duplicating this and fixing it up!
> 
> It looks like you are getting a CI failure at the mo since it is failing the parse the config. I think that's due to the final comma in the config causing issues.

Thanks for the review! Should be fixed now.

**Comment by bryphe** [comment]
Awesome, thank you @benox3 for the contribution (and thanks @CrossR for reviewing!)

**Pull request merged**